packages/e/efsw/xmake.lua

```diff
@@ -0,0 +1,44 @@
+package("efsw")
+    set_homepage("https://github.com/SpartanJ/efsw")
+    set_description("efsw is a C++ cross-platform file system watcher and notifier.")
+    set_license("MIT")
+
+    set_urls("https://github.com/SpartanJ/efsw/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/SpartanJ/efsw.git")
+    add_versions("1.1.0", "a67566d642510e3f571cc0f98d520bd806150362a51cfbc47366ed6c9890722f")
+
+    add_deps("cmake", { private=true })
```

Comment:
We don't need to add `{private}` for now, and I haven't figured out how to improve this.